gulpfile.babel.js

```diff
@@ -1,3 +1,6 @@
+// eslint-disable-next-line import/no-unassigned-import
+import 'babel-polyfill'
```

Comment:
polyfills are not included in the built library code right?

Reply:
No, this is only for gulp.